stock_storage_type_buffer/templates/stock_location_storage_buffer_templates.xml

```diff
@@ -0,0 +1,49 @@
+<odoo>
+    <template id="storage_buffer_help_message">
+        <p>The buffer locations
+            <ul class="mt8">
+                <t t-foreach="buffer_locations" t-as="loc_text">
+                    <li>
+                        <span itemprop="name" t-esc="loc_text" />
+                    </li>
+                </t>
+                <t t-if="buffer_locations_remaining">
+                    <t t-set="remaining" t-value="buffer_locations_remaining" />
```

Comment:
or
```xml
<t t-call="stock_storage_type_buffer.storage_buffer_help_message_other_locations">
    <t t-set="remaining" t-value="buffer_locations_remaining" />
</t>
```
not blocking, of course

Reply:
Oh nice! so much better indeed, fixed and squashed